### PR TITLE
Dev Meeting Todos

### PR DESCRIPTION
**Pull request by papr**

### CODE REVIEW

File: src/core/terminology.md [hunk 1]
```diff
@@ -5,19 +5,126 @@ permalink: /core/terminology
...
+in combination with other terms:
+
+- **World Camera**: Physical scene camera that captures the subject's field of view.
+- **World Process**: Application process responsible for processing the _world camera_
+    video stream and [_gaze mapping_](#gaze-positions).
+- **World Window**: The main Pupil Capture window; previews the result of the _world process_
+- **World Coordinate System**: Refers to the [_Coordinate System_](#coordinate-system) of the _world camera_.
 
 ## Eye
+The term `eye` is always used in combination with a different term that sppecifies the context:
```
Comment (willpatera) [inline src/core/terminology.md]: Typo

File: src/core/terminology.md [hunk 2]
```diff
@@ -5,19 +5,126 @@ permalink: /core/terminology
...
 
 ## Confidence
 
+Quality assessment of the _pupil detection_ for a given _eye_ image.
+- `0.0` means that the pupil could not be detected.
+- `1.0` highest possible value; the pupil was detected with very high certainty.
+
+_Gaze data_ inherits the _confidence_ value from the _pupil data_ it was based on.
+
 ## Calibration
```
Comment (willpatera) [inline src/core/terminology.md]: This can be explained in a simpler way for general audience IMO.

File: src/core/terminology.md [hunk 3]
```diff
@@ -5,19 +5,126 @@ permalink: /core/terminology
...
+
+During the _calibration_ process the subject is asked to fixate a list of target
+(reference) locations within the _world camera_'s field of view. The Pupil Core software
+collects these reference locations and _pupil data_ during the calibration. Afterwards,
+it correlates them in time and calculates a mapping function that is used to estimate
+the gaze for future pupil data.
+
+## Timing
+
+The Pupil Core software uses timestamps with 64-bit floating point values. Their unit is
```
Comment (willpatera) [inline src/core/terminology.md]:
```suggest
Pupil Core software uses timestamps with 64-bit floating point values. Their unit is
```

File: src/core/software/recording-format.md [hunk 4]
```diff
@@ -4,24 +4,93 @@ permalink: /core/software/recording-format
...
 
 ## Pupil Core 
 
-```
-list
-of
-files
-in
-recording
+```python
```
[the remark this answers is not in the document]
Comment (papr) [reply src/core/software/recording-format.md]: The syntax highlighting is nice. So only for visual/aesthetic reasons.

File: src/core/terminology.md (hunk 2 above)
Comment (papr) [reply src/core/terminology.md]: I am failing at finding an other abstraction level, that is not just `/core/#_4-calibration`... Did you have a specific simplification in mind?

File: src/developer/core/network-api.md [hunk 5]
```diff
@@ -54,46 +68,67 @@ you can send simple text messages to control Pupil Capture and Pupil Service fun
...
 ## IPC Backbone
 
-The IPC Backbone, the second stage of the Network API, grants you realtime access to
-nearly all data generated by Pupil Capture and Pupil Service.
+The IPC Backbone grants you realtime access to nearly all data generated by Pupil
+Capture and Pupil Service. It uses ZeroMQ's [PUB-SUB](http://zguide.zeromq.org/page:all#Getting-the-Message-Out)
+pattern for one-to-many communication.
 
-If you want to tap into the IPC backbone you will not only need the IP address but also
+If you want to tap into the IPC Backbone you will not only need the IP address but also
```
Comment (willpatera) [inline src/developer/core/network-api.md]:
```suggestion
If you want to tap into the IPC Backbone you will need both the IP address and the session's unique port.
```

File: src/developer/core/network-api.md [hunk 6]
```diff
@@ -54,46 +68,67 @@ you can send simple text messages to control Pupil Capture and Pupil Service fun
...
 the session's unique port. You can request them from `Pupil Remote`:
```
Comment (willpatera) [inline src/developer/core/network-api.md]:
```suggestion
 You can request them from `Pupil Remote`:
```

**Pull request merged**